components/apimgt/org.wso2.carbon.apimgt.rest.api.admin.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/admin/v1/utils/mappings/KeyManagerMappingUtil.java

```diff
@@ -210,6 +220,15 @@ public static KeyManagerConfigurationDTO toKeyManagerConfigurationDTO(String ten
         keyManagerConfigurationDTO.setOrganization(tenantDomain);
         keyManagerConfigurationDTO.setTokenType(keyManagerDTO.getTokenType().toString());
         keyManagerConfigurationDTO.setAlias(keyManagerDTO.getAlias());
+        KeyManagerPermissionsDTO permissions = keyManagerDTO.getPermissions();
+        if(permissions != null && permissions.getPermissionType() != null) {
```

Comment:
Formatting

Reply:
This might help. https://medium.com/@tharika/how-to-setup-code-formatting-in-my-ide-86374ec8fbd2

Reply:
Fixed in 1d58204afd0d2b23f33e003063649b697cf2e32a